crates/rbuilder/src/primitives/mod.rs

```diff
@@ -1,3 +1,5 @@
+//! Order types used as units of block building.
```

Comment:
"as units" is a bit strange to me.. "as elements of"? or "Order types used in block building"?